ChangeLog.md

```diff
@@ -4,21 +4,176 @@ Changelog
 Notable changes to the FreeOrion project will be documented in this file.
 
 
-[v0.4.9+] - TBA
+[v0.4.10] - TBA
 ---------------
 
+### Key Changes
+
+- Python 3.5+ is now supported and required. Python 2 dependence is removed
+- Added a new ship weapon part, the Arc Disruptor, which is useful for countering many cheap "chaff" ships
```

Comment:
Added a new ship weapon tech line, the Arc Disruptor, which is a good option for bad pilots, against comsats, against ships without shields, and has powerful part upgrades.